template.tpl

```diff
@@ -133,9 +195,11 @@ var getRdt = function() {
   return copyFromWindow('rdt');
 };
 
+var advancedParams = data.advancedMatchingParams && data.advancedMatchingParams.length ? makeTableMap(data.advancedMatchingParams, 'name', 'value') : {};
+
 var _rdt = getRdt();
 if (!_rdt.advertiserId) {
-  _rdt('init', data.id);
+  _rdt('init', data.id, advancedParams);
```

Comment:
Can you add `integration: 'gtm'` to the initialization parameters?

Reply:
Ah yeah, I had this change ready in my workspace but thought I would hold off until the pixel library changes were released. Although adding it here now or not would not really make a difference.

Reply:
I forgot I had this PR open: https://github.com/monicating/reddit-gtm-template/pull/1/files

Reply:
Ah, ok.  Cool!  Sounds good to me.  I didn't know if a merge here kicked off an automatic update to teh Google.  TY!